CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.11.0 (2023-05-30)
+
+- \_Breaking change:\_Provide the Bunny connection option `recovery_attempts` in Ears configuration.
```

Comment:
this doesn't seem to format correctly. Let's use `- **[BREAKING]** ` to hightlight breaking changes

```suggestion
- **[BREAKING]** Provide the Bunny connection option `recovery_attempts` in Ears configuration.
```